packages/components/src/components/hds/flyout/index.ts

```diff
@@ -59,7 +59,7 @@ export interface HdsFlyoutSignature {
 
 export default class HdsFlyout extends Component<HdsFlyoutSignature> {
   @tracked private _isOpen = false;
-  private _element!: HTMLDialogElement;
+  element!: HTMLDialogElement;
```

Comment:
I suggest we add a follow-up action for this.

```suggestion
  // TODO: make this property private; currently blocked by our consumers relying on it despite not being part of the public API: https://github.com/hashicorp/cloud-ui/blob/main/engines/waypoint/addon/components/preview-pane.ts#L15
  // private _element!: HTMLDialogElement;
  element!: HTMLDialogElement;
```